spec/features/trainees/edit_contact_details_spec.rb

```diff
@@ -0,0 +1,62 @@
+require "rails_helper"
+
+feature "edit contact details", type: :feature do
+  scenario "edit with valid parameters" do
+    given_a_trainee_exists
+    when_i_visit_the_contact_details_page
+    and_i_enter_valid_parameters
+    then_i_am_redirected_to_the_summary_page
+    and_the_contact_details_are_updated
+  end
+
+  scenario "changing locale clears previous address" do
+    given_a_trainee_exists
+    when_i_visit_the_contact_details_page
+    and_i_enter_an_international_address
+    then_i_am_redirected_to_the_summary_page
+    and_the_old_address_is_cleared
+  end
+
+  def given_a_trainee_exists
+    @trainee = create(:trainee, locale_code: :uk)
```

Comment:
```suggestion
    @trainee = create(:trainee)
```

Factory defaults it to uk